integration-tests/src/test/java/com/datastax/oss/driver/core/session/RemovedNodeIT.java

```diff
@@ -43,17 +46,23 @@
   public void should_signal_and_destroy_pool_when_node_gets_removed() {
     RemovalListener removalListener = new RemovalListener();
     try (CqlSession session = CqlSession.builder().withNodeStateListener(removalListener).build()) {
-      CCM_RULE.getCcmBridge().nodetool(2, "decommission");
+      assertThat(session.getMetadata().getTokenMap()).isPresent();
```

Comment:
This is to also test the opposite effect, that is, to assert that the token map is correctly shrank when a node is decommissioned.